src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Content/OrganizationListPage.xaml

```diff
@@ -36,126 +41,140 @@
-								   Visibility="{Binding Organizations.Result, Converter={StaticResource EnumerableHasAnyToCollapsed}, FallbackValue=Collapsed}"
-								   Style="{StaticResource Typo15}"
-								   Margin="16,12,16,0" />
-
-						<!-- About Message -->
-						<TextBlock Text="Learn more about this app and get source code at"
-								   Style="{StaticResource Typo13}"
-								   HorizontalAlignment="Center"
-								   TextAlignment="Center"
-								   TextWrapping="Wrap"
-								   Margin="16,8"
-								   Grid.Row="1" />
-
-						<!-- Hyperlink to website -->
-						<HyperlinkButton Content="About Us Page"
-										 Command="{Binding ToAboutPage}"
-										 HorizontalAlignment="Center"
-										 Margin="16,0,16,24"
-										 Grid.Row="2" />
+											  Margin="0,12" />
+
+								<!-- Verifying Message -->
+								<TextBlock Text="Loading..."
+										   Style="{StaticResource Typo08}" />
+							</StackPanel>
+						</Grid>
+
+						<!-- Error State -->
+						<Grid Visibility="{Binding Organizations.IsFaulted, Converter={StaticResource TrueToVisible}, FallbackValue=Collapsed}">
+
+							<!-- Message -->
+							<TextBlock Text="{Binding Organizations.IsInternetFaulted, Converter={StaticResource BoolToCustomString}}"
+									   Style="{StaticResource Typo10}"
+									   HorizontalAlignment="Center"
+									   VerticalAlignment="Center"
+									   TextAlignment="Center"
+									   Margin="50,0,50,48" />
+
+							<!-- Reload button -->
+							<Button x:Name="ReloadButton"
+									Content="Reload"
+									Command="{Binding ReloadPage}"
+									Style="{StaticResource ReloadButtonStyle}"
+									VerticalAlignment="Bottom"
+									Margin="16,22" />
+						</Grid>
+
+						<!-- Content -->
+						<Grid Visibility="{Binding Organizations.IsSuccess, Converter={StaticResource TrueToVisible}, FallbackValue=Collapsed}"
+							  Background="{StaticResource Color02Brush}">
+							<Grid.RowDefinitions>
+								<RowDefinition Height="*" />
+								<RowDefinition Height="Auto" />
+								<RowDefinition Height="Auto" />
+							</Grid.RowDefinitions>
+
+							<!-- Organizations List -->
+							<ListView ItemsSource="{Binding Organizations.Result}"
+									  VerticalAlignment="Top"
+									  IsItemClickEnabled="True"
+									  ItemClick="ListView_ItemClick">
+								<ListView.ItemTemplate>
+									<DataTemplate>
+										<Grid Height="44">
+
+											<!-- Organization Name -->
+											<TextBlock Text="{Binding AccountName}"
+													   Style="{StaticResource Typo08}"
+													   TextWrapping="NoWrap"
+													   VerticalAlignment="Center"
+													   Margin="16,0,50,0" />
+
+											<!-- Border -->
+											<Rectangle Fill="{StaticResource Color07Brush}"
+													   Height="1"
+													   VerticalAlignment="Bottom"
+													   Margin="16,0,0,0" />
+
+											<!-- Chevron -->
```

Comment:
Is this used anywhere else? It could be in a PathControl.xaml ResourceDictionary

Reply:
Not related to this PR but will look at it :)